src/components/controls/PasswordInput/README.md

```diff
@@ -0,0 +1,39 @@
+## Password Input
+
+```tsx
+import {PasswordInput} from '@gravity-ui/uikit';
+```
+
+Password Input display component
+
```

Comment:
Let's also make separate sections `Reveal Button` and `Copy Button` describing behavior and props to enable it